src/three-sixty-viewer.js

```diff
@@ -136,6 +140,13 @@ export default class ThreeSixtyViewer {
       this.animationFrameId = requestAnimationFrame(loop);
       let cameraUpdated = this.controls.update();
       this.renderer.render(this.scene, this.camera, this.needsUpdate || cameraUpdated);
+      if(this.viewDirectionChangeCb) {
+        if(this.viewDirectionChangeLastPhiSent !== this.controls.phi || this.viewDirectionChangeLastThetaSent !== this.controls.theta) {
```

Comment:
I think you can rely on the `cameraUpdated` for this? its value is`true` only if the camera moved. Then you could just check if `this.viewDirectionChangeCb && cameraUpdated` in the first if